src/frontend/sharedComponents/icons/Circle.js

```diff
@@ -2,28 +2,42 @@
 import * as React from "react";
 import { StyleSheet, View } from "react-native";
 import type { ViewStyleProp } from "../../types";
+import validateColor from "validate-color";
 
 type CircleProps = {
+  color?: string,
+  borderWidth?: number,
   radius?: number,
   children: React.Node,
   style?: ViewStyleProp,
 };
 
-const Circle = ({ radius = 25, style, children }: CircleProps) => (
-  <View
-    style={[
-      styles.circle,
-      {
-        width: radius * 2,
-        height: radius * 2,
-        borderRadius: radius * 2,
-      },
-      style,
-    ]}
-  >
-    {children}
-  </View>
-);
+const Circle = ({
+  color,
+  borderWidth = 2,
+  radius = 25,
+  style,
+  children,
+}: CircleProps) => {
+  const validColor = color && validateColor(color);
+  return (
+    <View
+      style={[
+        styles.circle,
+        {
+          borderWidth: validColor && borderWidth,
+          borderColor: validColor && color,
```

Comment:
non-blocking but technically these properties don't accept a potential boolean - guessing that React Native handles this okay but if this component were written in TS, the linter may complain about this 😄 if it rendered okay when testing it out, wouldn't worry about fixing this right now - can be done whenever we migrate this component to TS

Reply:
update on this: this turns out to be a blocking issue. Suggestion to remove the `borderColor` and `borderWidth` values in the `style` object and update this section to be:

```suggestion
          borderWidth: validColor ? 1.5 : 1,
          borderColor: validColor ? color : "#EAEAEA",
```

Reply:
The suggested `borderWidth` adjustment needs to be amended because of https://github.com/digidem/mapeo-mobile/pull/1004#discussion_r976849914